### DISCUSSION TITLE
When 2.1.0 release is planned to be available?

### DISCUSSION THREAD

**Discussion by ydrozhdzhal** [apolloconfig/apollo#4573] [General]
Hi

Can you provide information about when do you plan to release 2.1.0 and upload artifacts to maven central?

Thank you

**Comment by nobodyiam** [comment]
There is no specific plan now, but I think it would be late this year or early 2023 if there is no urgent need.

**Reply by ydrozhdzhal** [reply]
We are using Apollo in our application and now there is urgent need to have this feature: https://github.com/apolloconfig/apollo/pull/4409

Is it possible to have this released soon, for example, as 2.0.2 and made it available in Maven central repo?

**Reply by nobodyiam** [reply]
There is a discussion to move some modules out of the main repository(#4427). I think we could release the apollo-client in a different release cycle when it's moved to a standalone repository.

**Reply by ydrozhdzhal** [reply]
For us it would be noce to have new client released ASAP.

**Reply by ydrozhdzhal** [reply]
It would be nice if you'll ping me here in case new release would be available in maven repo.

**Reply by nobodyiam** [reply]
Sure, we'll do that.

**Reply by nobodyiam** [reply]
The apollo-client 2.1.0 is now available in the [maven central repo](https://mvnrepository.com/artifact/com.ctrip.framework.apollo/apollo-client).

**Reply by ydrozhdzhal** [reply]
Thank you for notification.
